consensus/consensus-types/src/common.rs

```diff
@@ -39,14 +39,23 @@ impl Payload {
     pub fn len(&self) -> usize {
         match self {
             Payload::DirectMempool(txns) => txns.len(),
-            Payload::InQuorumStore(_poavs) => todo!(),
         }
     }
 
     pub fn is_empty(&self) -> bool {
         match self {
             Payload::DirectMempool(txns) => txns.is_empty(),
-            Payload::InQuorumStore(_poavs) => todo!(),
+        }
+    }
+
+    /// This is computationally expensive on the first call
```

Comment:
Is this true? The get_batch call on the mempool side also calls raw_txn_bytes_len

Reply:
this is on receiver side, I think it's the first time to call that. the next time is in execution I think